module/resolve.go

```diff
@@ -195,10 +195,10 @@ func (r *remoteResolver) Resolve(ctx context.Context, scope *parser.Scope, decl
 		return nil, err
 	}
 
-	var solveOpts []solver.SolveOption
-	for id, path := range cg.Locals {
-		solveOpts = append(solveOpts, solver.WithLocal(id, path))
-	}
+	// var solveOpts []solver.SolveOption
```

Comment:
remove?

Reply:
Note that `(*CodeGen).GenerateImport` still needs to return `llb.State` because we need to generate the vertex digest for the import.

Reply:
I addressed it here:
- https://github.com/openllb/hlb/pull/49/commits/668cb87ff69c5cc0500c82c89da53a894b3493c2#diff-5465bec959c3455c1a373bcf1569ad5a
- https://github.com/openllb/hlb/pull/49/commits/668cb87ff69c5cc0500c82c89da53a894b3493c2#diff-641e469b8b7432439f761283de3bfd0c